packages/dev/loaders/src/glTF/2.0/Extensions/interactivityPathCameraExtensions.ts

```diff
@@ -0,0 +1,84 @@
+import type { Camera } from "core/Cameras/camera";
+import type { FlowGraphPath } from "core/FlowGraph/flowGraphPath";
+import type { FlowGraphContext } from "core/FlowGraph/flowGraphContext";
+import { Tools } from "core/Misc/tools";
+import type { IGLTF } from "../glTFLoaderInterfaces";
+
+const camerasRegex = /^\/cameras\/(\d+)\/(orthographic|perspective)\/(xmag|ymag|zfar|znear|aspectRatio|yfov)$/;
+
+function getBabylonCamera(path: FlowGraphPath, context: FlowGraphContext): { babylonCamera: Camera; gltfProperty: string } {
+    const fullPath = path.getFinalPath();
+    const gltfTree = context.getVariable("gltf") as IGLTF;
+    if (!gltfTree) {
+        throw new Error(`No glTF tree found for path ${fullPath}`);
+    }
+    const matches = fullPath.match(camerasRegex);
+    if (!matches || matches.length !== 4) {
+        throw new Error(`Invalid path ${fullPath}`);
+    }
+    const cameraIndex = parseInt(matches[1]);
+    const camera = gltfTree.cameras && gltfTree.cameras[cameraIndex];
+    if (!camera) {
+        throw new Error(`Invalid camera index for path ${fullPath}`);
+    }
+    const babylonCamera = (camera as any)._babylonCamera;

```

Comment:
Why do you need to cast to `any`?

Reply:
I wasn't importing from loaderInterfaces before so it didn't have the property, but I can remove the any now, good catch